python_binding/lsm/_local_disk.py

```diff
@@ -420,3 +420,34 @@ def led_status_get(disk_path):
                 No capability required as this is a library level method.
         """
         return _use_c_lib_function(_local_disk_led_status_get, disk_path)
+
+    @staticmethod
+    def link_speed_get(disk_path):
+        """
+        Version:
+            1.4
+        Usage:
+            Get link speed for specified disk.
+        Parameters:
+            disk_path (string)
+                The disk path, example '/dev/sdb'.
+        Returns:
+            link_speed
+                String for link speed. The string is like '3.0 Gbps'.
+        SpecialExceptions:
+            LsmError
+                ErrorNumber.LIB_BUG
+                    Internal bug.
+                ErrorNumber.INVALID_ARGUMENT
+                    Invalid disk_path. Should be like '/dev/sdb'.
+                ErrorNumber.NOT_FOUND_DISK
+                    Provided disk is not found.
+                ErrorNumber.NO_SUPPORT
+                    Provided disk does not support SCSI SPC.
```

Comment:
Perhaps misleading to say that this means the SCSI Spec is unsupported? Is there a more generic way to communicate that the disk does not support the spec appropriate for whatever type of device it is?